dev/cuda/classifier_fused.cu

```diff
@@ -169,6 +259,13 @@ void fused_classifier(int kernel_num, float* dlogits, float* losses,
         case 1:
             fused_classifier1(dlogits, losses, logits, dlosses, targets, B, T, V, block_size);
             break;
+        case 2:
+            if((V % 4) != 0) {
```

Comment:
I'd move this inside `fused_classifer2` at the top?

Reply:
Removed it after ngc92's changes that split V and P (padded V) so it's implicitly guaranteed by the function declaration.